spec/unit/multi_select_spec.rb

```diff
@@ -905,4 +909,48 @@ def exit_message(prompt, choices)
       expect(prompt.output.string).to eq(expected_output)
     end
   end
+
+  context "with :min and :max" do
+    let(:choices) { %w[A B C] }
+    before do
+      prompt.on(:keypress) do |e|
+        prompt.trigger(:keydown) if e.value == "j"
+      end
+    end
+    context "with min <= max" do
+      let(:min) { 2 }
+      let(:max) { 2 }
+      it "requires a min number of choices" do
+        keydown = "j"
+        spacebar = " "
+        enter = "\r"
```

Comment:
Let's follow the other tests and inline these as inputs. I don't think it will distract from test meaning if we do:

```ruby
prompt.input << " " << "\n" << "j" << " " << "\n"
```